site/content/contribute/mvp/_index.md

```diff
@@ -13,8 +13,9 @@ Hundreds of developers around the world contribute to Mattermost open source pro
 The title of “Most Valued Professional” is awarded to an outstanding contributor for a key Mattermost release.
 
 
-Version |	   Release date     | MVP
+Version |	   Release date   | MVP
```

Comment:
That line seams messed up. Would you please remove the tab?

Reply:
Which tab?

Reply:
this is ok, makes no difference in the rendered page